### PR TITLE
Commands: Add Github related commands

### PR DESCRIPTION
**Pull request by IdanHo**
This also renames the discord token environment variable to "discord_token" and adds an optional github token environment variable ("github_token") that allows the bot to have a higher rate limit by linking it against a github account. (https://octokit.github.io/rest.js/v18#authentication)

### CODE REVIEW

**Review by kleinesfilmroellchen — APPROVED** [review]

File: src/commands/prCommand.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    help(commandPrefix: string): string {
+        return `Use ${commandPrefix}pr <keywords> to search for SerenityOS pull requests`;
+    }
+
+    async run(parsedUserCommand: CommandParser): Promise<void> {
+        const result = await githubAPI.search_pull_requests(parsedUserCommand.args.join("+"));
+        if (result) {
+            await parsedUserCommand.send(`${result.html_url}`);
+        } else {
+            await parsedUserCommand.send(`No matching pull requests found :(`);
```
Comment (kleinesfilmroellchen) [inline src/commands/prCommand.ts:24]: even better: `:^(`

File: src/models/commandParser.ts [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 export class CommandParser {
-    readonly parsedCommandName: string; /** Command name in all lowercase. */
-    readonly args: string[]; /** Arguments (split by space). */
-    readonly originalMessage: Message; /** Original Message the command was extracted from. */
+    readonly parsedCommandName: string;
+    /** Command name in all lowercase. */
+    readonly args: string[];
+    /** Arguments (split by space). */
+    readonly originalMessage: Message;
+    /** Original Message the command was extracted from. */
```
Comment (kleinesfilmroellchen) [inline src/models/commandParser.ts:10]: This appears to be a linter error: the comments should be before the argument they describe

Comment (IdanHo) [reply src/models/commandParser.ts:9]: yep, fixed

File: src/commands/prCommand.ts (hunk 1 above)
Comment (IdanHo) [reply src/commands/prCommand.ts]: good suggestion :^)

**Comment by bgianfo** [comment]
This is great! I just updated the deployment with the new config values... so lets see if this works :)

**Pull request merged**